packages/dataviews/src/layouts.ts

```diff
@@ -37,3 +38,29 @@ export const VIEW_LAYOUTS = [
 		icon: isRTL() ? formatListBulletsRTL : formatListBullets,
 	},
 ];
+
+export function getMandatoryFields( view: View ): string[] {
+	if ( view.type === 'table' ) {
+		return [ view.layout?.primaryField ]
+			.concat(
+				view.layout?.combinedFields?.flatMap(
```

Comment:
Why should we consider combined fields as mandatory? 🤔

Reply:
Because we don't have the right UI and components to actually disable them. If you manage to make them work without issues, you can remove this condition :)